configs/_base_/visualization/kitti-3d-3class.py

```diff
@@ -0,0 +1,42 @@
+# dataset settings
+dataset_type = 'KittiDataset'
```

Comment:
Suggest using the configs in _base_.
To skip the augmentation pipeline, we can write a function to skip augmentation or only keep loading pipeline like https://github.com/open-mmlab/mmdetection/blob/eaf42fa3a1c826e6d8e9af7a5207a26f1a04d636/mmdet/datasets/utils.py#L73